src/main/java/com/snowflake/kafka/connector/internal/LoggerHandler.java

```diff
@@ -37,19 +27,41 @@ public static void setConnectGlobalInstanceId(UUID id) {
    *
```

Comment:
nit: `remove If an invalid id is given...`

Reply:
removed logging logic, resolving any related comments